### PR TITLE
fixed RH_PASSWORD variable to be compatible with passwords containing…

### PR DESCRIPTION
**Pull request by Nick-Harvey**
… special characters

### CODE REVIEW

**Comment by cooktheryan** [comment]
@glennswest would this be applicable to Azure as well

**Comment by glennswest** [comment]
I've added this support in, nice catch.

**Comment by cooktheryan** [comment]
This LGTM

**Comment by cooktheryan** [comment]
Hate to string this along PR much longer but @dav1x and @markllama can you see if your providers are affected by this as well. I will be testing AWS tomorrow but I believe I face the same issue.

File: reference-architecture/gce-cli/gcloud.sh [hunk 1]
```diff
@@ -367,7 +367,7 @@ if ! gcloud --project "$GCLOUD_PROJECT" compute images describe "$REGISTERED_IMA
         sleep 5
     done
     if ! gcloud -q --project "$GCLOUD_PROJECT" compute ssh "cloud-user@${TEMP_INSTANCE}" --zone "$GCLOUD_ZONE" --ssh-flag="-t" --command "sudo bash -euc '
-        subscription-manager register --username=${RH_USERNAME} --password=${RH_PASSWORD};
+        subscription-manager register --username=${RH_USERNAME} --password="\"${RH_PASSWORD}\"";
```
Comment (pschiffe) [inline reference-architecture/gce-cli/gcloud.sh]: @Nick-Harvey thanks for a catch and a PR, but shouldn't be `--password=\"${RH_PASSWORD}\";` enough? Could you check and possibly, update the PR? Thanks.

**Comment by Nick-Harvey** [comment]
You're right. It should just be `password=\"${RH_PASSWORD}\"` the additional `"` were artifacts from some of my earlier troubleshooting. I'll make the changes and retest to make 100% sure that it works and then resubmit pull request. Standby

**Pull request merged**